front50-sql/src/main/kotlin/com/netflix/spinnaker/front50/model/SqlStorageService.kt

```diff
@@ -258,63 +258,58 @@ class SqlStorageService(
   }
 
   override fun <T : Timestamped> storeObjects(objectType: ObjectType, allItems: Collection<T>) {
-    // using a lower `chunkSize` to avoid exceeding default packet size limits.
-    allItems.chunked(100).forEach { items ->
-      try {
-        withPool(poolName) {
-          jooq.transactional(sqlRetryProperties.transactions) { ctx ->
+    withPool(poolName) {
```

Comment:
With this flip, I think the entire batch fails now vs. only a chunk of it - intentional?

Reply:
Guess that's partly the point of this PR - just wondering if that's a good thing?

Reply:
Can ignore this mostly just curious philosophically which is better :)